systests/microprofile/client/async/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0"?>
+<!--
+  ~  Licensed to the Apache Software Foundation (ASF) under one
+  ~  or more contributor license agreements.  See the NOTICE file
+  ~  distributed with this work for additional information
+  ~  regarding copyright ownership.  The ASF licenses this file
+  ~  to you under the Apache License, Version 2.0 (the
+  ~  "License"); you may not use this file except in compliance
+  ~  with the License.  You may obtain a copy of the License at
+  ~
+  ~  http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing,
+  ~  software distributed under the License is distributed on an
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>cxf-microprofile-tck</artifactId>
+        <groupId>org.apache.cxf.systests</groupId>
+        <version>3.3.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.cxf.systests</groupId>
+    <artifactId>cxf-microprofile-async</artifactId>
+    <name>Apache CXF MicroProfile Async Sys Tests</name>
+    <description>Apache CXF System Tests - MicroProfile Rest Client Async Tests</description>
+    <url>http://cxf.apache.org</url>
+    <properties>
+        <cxf.geronimo.config.version>1.0</cxf.geronimo.config.version>
```

Comment:
Could you please add the `cxf.module.name` to the `<properties>` section:
```
        <cxf.module.name>org.apache.cxf.systests.microprofile.async</cxf.module.name>
```
Thanks!

Reply:
@reta done.  thanks for the review.

Reply:
:+1: thanks!